content/en/network_monitoring/devices/vpn_monitoring.md

```diff
@@ -0,0 +1,134 @@
+---
+title: VPN Monitoring
+description: Get started with monitoring your devices VPN tunnels.
+further_reading:
+  - link: "https://www.datadoghq.com/knowledge-center/network-monitoring/snmp-monitoring/"
+    tag: "Knowledge Center"
+    text: "SNMP Monitoring Overview"
+  - link: "https://www.datadoghq.com/blog/monitor-snmp-with-datadog/"
+    tag: "Blog"
+    text: "Monitor SNMP with Datadog"
+---
+
+## Overview
+
+VPN monitoring provides visibility into your devices VPN tunnels within Network Device Monitoring (NDM). This feature allows you to keep track of critical data and metrics to examine the health of your VPN tunnels.
+
+When configured, the SNMP check collects VPN tunnel data from your devices. This can be configured for individual as well as Autodiscovered devices.
+
+### Limitations
+
+- Only Cisco IPsec VPN tunnels are currently supported.
+
+## Configuration
+
+1. Install or upgrade the [Datadog Agent][1] to v7.70+.
+
+2. Edit the `snmp.d/conf.yaml` file in the `conf.d/` folder at the root of your [Agent's configuration directory][2] for individual devices, or the [`datadog.yaml`][3] Agent configuration file for Autodiscovery.
+
+{{< tabs >}}
+{{% tab "Individual" %}}
+
+To enable VPN monitoring to all _manually_ configured devices, add the `collect_vpn` configuration in the `init_config` section:
+
+```yaml
+init_config:
+  loader: core
+  use_device_id_as_hostname: true
+  collect_vpn: true          # (default false) enable collecting VPN tunnel data
+instances:
+  - ip_address: '1.2.3.4'
+    community_string: 'sample-string'
+    tags:
+    - 'key1:val1'
+    - 'key2:val2'
+```
+
+Or, apply the `collect_vpn` configuration _per_ instance:
+
+```yaml
+init_config:
+  loader: core
+  use_device_id_as_hostname: true
+instances:
+  - ip_address: '1.2.3.4'
+    community_string: 'sample-string'
+    tags:
+      - 'key1:val1'
+      - 'key2:val2'
+    collect_vpn: true        # (default false) enable collecting VPN tunnel data
```

Comment:
```suggestion
    collect_vpn: true        # (default false) enable collecting VPN tunnel data
{{< /highlight >}}
```
Here is an example of how this works (Please double check my formatting):
https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown+Shortcode+Reference#Highlight-lines-in-code-blocks